pkg/resources/show_and_describe_handlers.go

```diff
@@ -43,6 +43,28 @@ type showMapping struct {
 	normalizeFunc  func(any) any
 }
 
+// handleExternalValueChangesToObjectInDescribe assumes that describe output is kept in DescribeOutputAttributeName attribute
+func handleExternalValueChangesToObjectInDescribe(d *schema.ResourceData, mappings ...describeMapping) error {
```

Comment:
this is copy-paste of handleExternalChangesToObjectInShow, only the var names and attribute name changes. Please extract this as a common body (it may be done in a separate PR)